integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_files_excluded.py

```diff
@@ -71,54 +71,19 @@ def test_multiple_granules_files_excluded_passes(self):
             }
 
             expected_output = {
-                "payload": {
-                    "granules": [
-                        {
-                            "granuleId": granule_id,
-                            "createdAt": createdAt_time,
-                            "files": [
-                                {
-                                    "bucket": bucket_name,
-                                    "key": key_name
-                                }
-                            ]
-                        }
-                    ],
-                    "copied_to_orca": []
-                },
-                "meta": {
-                    "provider": {
-                        "id": provider_id,
-                        "name": provider_name
-                    },
-                    "collection": {
-                        "meta": {
-                            "orca": {
-                                "excludedFileExtensions":
-                                    excluded_filetype
+                "granules": [
+                    {
+                        "granuleId": granule_id,
+                        "createdAt": createdAt_time,
+                        "files": [
+                            {
+                                "bucket": bucket_name,
+                                "key": key_name
                             }
-                        },
-                        "name": collection_name,
-                        "version": collection_version
+                        ]
                     }
-                },
-                "cumulus_meta": {
-                    "execution_name": execution_id
-                },
```

Comment:
so the output `step_function_results["output"]` doesn't show these keys?

Reply:
Nope. Automatically added by CMA to pass to future events. We can just use the step-function variables since we aren't pulling from s3.

Reply:
Nope. Good catch, I'll update the relevant docs.